src/App.tsx

```diff
@@ -36,4 +36,8 @@ const PrivateRoute = ({ component: Component, ...rest }: any) => (
     />
 );
 
+const LoginRoute = ({ component: Component, ...rest }: any) => (
+    <Route {...rest} render={(props) => (AuthUtil.checkAuth() ? <Component {...props} /> : <Login {...props} />)} />
```

Comment:
Why did we go with this form of displaying the dashboard instead of `Redirect to={{ pathname: 'dashboard' }}` like we do in `PrivateRoute`? The disadvantage with the current way is that the URL says login despite being at the dashboard.